src/features/quest/ui/styles.ts

```diff
@@ -0,0 +1,246 @@
+import styled, { css } from 'styled-components';
+import { getImageUrl } from '@utils/getImageUrl';
+import { media } from '@/style/media';
+
+interface QuestUrlProps {
+  $isLearn?: boolean;
+  $isQuest?: boolean;
+}
+
+export const QuestContainer = styled.div<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      width: 270px;
+      margin-right: 28px;
+
+      ${media.mobile} {
+        display: none;
+      }
+    `}
+
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+      width: 683px;
+      margin-top: 40px;
+    `}
+`;
+
+// 퀘스트 섹션
+export const QuestSection = styled.section<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      width: 270px;
+      height: 210px;
+      margin: 89px 0 0 0;
+      position: fixed;
+      background-image: url(${getImageUrl('종이.svg')});
+    `}
+
+  ${({ $isQuest = true }) => $isQuest && css``}
+`;
+
+// QuestContent와 QuestItemsContainer를 감싸는 컨테이너
+export const QuestContentWrapper = styled.div<QuestUrlProps>`
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: flex-start;
+    `}
+`;
+
+// '오늘의 퀘스트' 텍스트 + 아이콘 감싸는 div
+export const QuestContent = styled.div<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin-bottom: 15px;
+    `}
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+      width: 683px;
+      border-radius: 10px;
+      background: #ffb53d;
+      box-shadow: 0 5px #f09900;
+      height: 36px;
+    `}
+`;
+
+// '오늘의 퀘스트' 아이콘
+export const QuestIcon = styled.img<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      width: 38px;
+      height: 42px;
+      margin-bottom: 5px;
+    `}
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+      display: none;
+    `}
+`;
+
+// '오늘의 퀘스트' 혹은 '메인 퀘스트' 텍스트
+export const DailyOrMainQuestText = styled.p<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      font-size: 15px;
+      color: #9f9f9f;
+      text-align: center;
+      margin-top: 20px;
+    `}
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+      height: 36px;
+      color: #fff;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      font-size: 17px;
+      font-weight: 700;
+    `}
+`;
+
+export const QuestItemsContainer = styled.div<QuestUrlProps>`
+  ${({ $isLearn }) =>
+    $isLearn &&
+    css`
+      display: none;
+    `}
+
+  ${({ $isQuest = true }) =>
+    $isQuest &&
+    css`
+    width: 100%
+    height: 100%
+    display: flex;
+    flex-direction: column;
+  `}
```

Comment:
```suggestion
    css`
      width: 100%
      height: 100%
      display: flex;
      flex-direction: column;
  `}
```
이쪽 컨벤션이 다른 곳과 다른 것 같습니다 :)